bitbucket/request.js

```diff
@@ -130,7 +128,7 @@ module.exports = function Request(_options) {
       if (method === 'POST') {
         query = JSON.stringify(parameters);
         headers['Content-Type'] = 'application/json';
-        if (!options.use_xhr) {
+        if (!options.requesterFn) {
```

Comment:
I would remove this and just set the `Content-Type` blindly. If some particular implmentation (\*cough cough XHR cough\*) needs to delete that header from the requests it makes, that's _its_ problem. This should provide the requesterFn with all the standard headers it needs in the general case. This applies to `prepRequest` as well.